### ISSUE TITLE
Converted caffe model produce wrong result.

### ISSUE THREAD

**Issue by mariolew** [miaow1988/ShuffleNet_V2_pytorch_caffe#14]
If I don't load pre-trained pytorch model, the result of the converted model has diff within 0.001, but when I load the pre-trained model, the diff is very big. I don't know why, since the pre-trained model has been successfully loaded. BTW, the pretrained model is trained through multi-gpu.

**Comment by mariolew** [comment]
solved.
